x-pack/plugins/ml/public/jobs/jobs_list/components/multi_job_actions/group_selector/new_group_input/new_group_input.js

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+
+import PropTypes from 'prop-types';
+import React, {
+  Component,
+} from 'react';
+
+import {
+  EuiButtonIcon,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiFieldText,
+  EuiFormRow,
+} from '@elastic/eui';
+
+import './styles/main.less';
+import { validateGroupNames } from '../../../validate_job';
+import { KEY_CODES } from 'plugins/ml/../common/constants/key_codes';
```

Comment:
Would need to use a relative path when creating Jest tests, or use the eui `keyCodes` import instead